src/test/CMakeLists.txt

```diff
@@ -223,6 +223,11 @@ add_executable(kvstorebench
 target_link_libraries(kvstorebench librados global ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS}
   ${TCMALLOC_LIBS})
 
+# osbench
+add_executable(osbench objectstore_bench.cc
```

Comment:
@cbodley cool! seems we are deprecating autotools. =) but it would be ideal to at least see if `osbench` and `unittest_pageset` compile or not on supported platforms before dropping the support of the autotools.

BTW, if you add `osbench` to Makefile.am, please be sure to update `src/.gitignore` as well. otherwise the gitbuilder will be annoyed: the build script expects a "clean" repo.